### PR TITLE
Bug 1160981 - [Stingray][EPG] Basic HTML UI mockup

### PR DESCRIPTION
**Pull request by sean2449**

### CODE REVIEW

**Comment by autolander** [comment]
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=626d28cb47b386b59edc555fe09d264540a04936

**Comment by autolander** [comment]
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=322af74bd9931d51e4edbc6e21b0b38cd340caa9

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=9b592ec429c2ecbe67c82974a72d00b58b7889c6

File: tv_apps/tv-epg/jsdoc.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "smart-home": {
```
Comment (huchengtw-moz) [inline tv_apps/tv-epg/jsdoc.json]: We should rename this field.

File: tv_apps/tv-epg/manifest.webapp [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    }
+  },
+  "default_locale": "en-US",
+  "icons": {
+    "354": "/style/icons/ic_default_app.png"
+  },
+  "orientation": "landscape",
+  "permissions": {
+    "tv": {},
+    "webapps-manage": {},
```
Comment (huchengtw-moz) [inline tv_apps/tv-epg/manifest.webapp]: Why this permission?

I don't think we need them now. Please review `webapps-manage` and `systemXHR` permission. We should add them once we add the feature who requires them.

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=16292efa6d9211509465ca09f931a24e50b4b0d2

**Comment by autolander** [comment]
http://docs.taskcluster.net/tools/task-graph-inspector/#YU1dbvwMQFixUByKShFLHQ

The pull request failed to pass integration tests. It could not be landed, please try again.

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=39cd19b9e079c93b96f70c8942241b73c1f0d506

**Comment by autolander** [comment]
http://docs.taskcluster.net/tools/task-graph-inspector/#QvzsMYQgQ3SSTvQZsBxkAg

The pull request failed to pass integration tests. It could not be landed, please try again.

**Pull request merged**